Bottom: 9bd3c3f5df66b9bffcdf10a227ece36912d6dc69
Top:    8124068695760b7539d8c18ddaf41096b79e66a7
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-04-16 11:33:50 -0400

Refresh of mmc-dma.patch

---

diff --git a/drivers/mmc/host/lpc31xx_mmc.c b/drivers/mmc/host/lpc31xx_mmc.c
index adefd33..9ad5037 100644
--- a/drivers/mmc/host/lpc31xx_mmc.c
+++ b/drivers/mmc/host/lpc31xx_mmc.c
@@ -140,9 +140,6 @@ struct lpc313x_mci_slot {
 };
 
 /* Register access macros */
-//#define mci_readl(reg) __raw_readl(&SDMMC_##reg)
-//#define mci_writel(reg,value) __raw_writel((value),&SDMMC_##reg)
-
 static inline uint32_t mci_readl(struct lpc313x_mci *host, uint32_t reg)
 {
 	return __raw_readl(host->regs + reg);
